### ISSUE TITLE
new method : flush expired cache

### ISSUE THREAD

**Issue by mqu** [DannyBen/lightly#11]
hello Danny, would be nice to be able to remove expired keys files.

**Comment by DannyBen** [comment]
Yeah, should be easy. I am thinking maybe a `prune` method?

**Comment by DannyBen** [comment]
Added. Do you want to test it before I release? If so, this should be in your Gemfile:

```ruby
# Gemfile
source "https://rubygems.org"
git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }

gem "lightly", github: 'DannyBen/lightly', branch: 'prune'
```

And this is a simple test case:

```ruby
# test.rb
require 'lightly'

Lightly.life = '30s'
Lightly.get('key1') { 'content' }
sleep 30
Lightly.get('key2') { 'content' }
Lightly.prune
```

And then just run with `bundle exec ruby test.rb`

**Comment by DannyBen** [comment]
Released as 0.3.2.

**Comment by mqu** [comment]
thank you ; very good job.
